mitype/app.py

```diff
@@ -292,6 +293,9 @@ def update_state(self, win):
             self.start_time = 0
         win.refresh()
 
+    def make_curses_visible(self, make_visible):
+        curses.curs_set(1 if make_visible else 0)
+
```

Comment:
It would be very straight forward to write down cursor.set_curs(0) instead of the function you wrote. The function is just a layer over another function that is pretty self-explaining.